.github/workflows/linux.yml

```diff
@@ -0,0 +1,23 @@
+on:
```

Comment:
Please add the GPLv3 and Tokiwa-header and include a short description on what this is.